### PR TITLE
Rework OpenTelemetry APM UI pages

### PR DESCRIPTION
**Pull request by jack-berg**
Builds off #17169, #17210, #17269.

Rework and merge (almost) all the pages under to "OpenTelemetry -> Find data in New Relic" into new "OpenTelemetry -> OpenTelemetry data in New Relic -> OpenTelemetry APM UI". The only exceptions are "Create custom queries" and "Alerts". The content on these pages is not exclusive to OpenTelemetry APM, so moved them to be children under OpenTelemetry -> Find data in New Relic" for the time being. Will come back and rework them in a future PR.

Each page within the OpenTelemetry APM UI is given a top level header so you can navigate between them at a glance using the sidebar navigation. There are also a few topics which are cross cutting concerns (i.e. "Golden Signals", "Metrics or spans toggle"). These are also given top level headers.

### CODE REVIEW

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+Metrics are not subject to sampling, and are thus more accurate, especially when computing rates like throughput. However, metrics are subject to cardinality constraints, and may lack certain attributes important for analysis. In contrast, spans are sampled and thus subject to accuracy issues, but have richer attributes since they are not subject to cardinality constraints.
+
+Historically, OpenTelemetry language APIs / SDKs and instrumentation prioritized trace instrumentation. However, the project has come a long way and [metrics](https://opentelemetry.io/docs/languages/) are available in almost all languages. Check the documentation of the relevant language and instrumentation for more details.
+
+[http metric]: https://opentelemetry.io/docs/specs/semconv/http/http-metrics/
+[http span]: https://opentelemetry.io/docs/specs/semconv/http/http-spans/
+[rpc metric]: https://opentelemetry.io/docs/specs/semconv/rpc/rpc-metrics/
+[rpc span]: https://opentelemetry.io/docs/specs/semconv/rpc/rpc-spans/
+[service entity attributes]: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources/#services
```
Comment (ally-sassman) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx:171]: @jack-berg What's the purpose of these doc links? They should have some kind of preface beforehand, or be connected to the paragraph above

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-go-runtime-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-logs-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-errors-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-errors-inbox-page
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-metrics-explorer-page
+---
+
+import opentelemetryViewSpanEvents from 'images/opentelemetry_screenshot-crop_view-span-events.webp'
+
+The OpenTelemetry APM UI is a curated experience available for [service entities][service entity attributes]. See [set up your own app](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-set-up-your-app) for details on configuring services with OpenTelemetry APM monitoring. See [Find OpenTelemetry APM services](#find-opentelemetry-apm-services) to find OpenTelemetry APM entities in New Relic.
```
Comment (ally-sassman) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx:30]: @jack-berg Where is "[service entity attributes]" supposed to link to?

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx (hunk 1 above)
Comment (alanwest) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx:171]: These do not get rendered. It's a feature of markdown that enables you to create a reference to links that are used multiple times within the document.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx (hunk 2 above)
Comment (alanwest) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx:30]: This is a usage of the referenced link declared at the bottom of the document.

**Review by ghost — DISMISSED** [review]

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx [hunk 3]
```diff
@@ -87,85 +87,79 @@ The service map page provides a visual representation of your entire architectur
...
+For spans, the queries are generic, utilizing only the top level span data model. Spans are counted towards throughput and response time if they are root entry spans into a service, computed using a heuristic of `WHERE span.kind = server OR span.kind = consumer`. Spans are errors if they have a status code of `ERROR` (for example, `otel.status_code = ERROR`).
 
 ## Narrow data with filters [#narrow-with-filters]
 
-Several pages include a filter bar (i.e. **Narrow data to..**). This allows you to filter queries on the page to match the criteria. For example, you might narrow to a particular canary deployment by filtering for `service.version='1.2.3-canary'`. Filters are preserved when navigating between pages.
+Several pages include a filter bar, with options like **Narrow data to...**. This allows you to filter queries on the page to match the criteria. For example, you might narrow to a particular canary deployment by filtering for `service.version='1.2.3-canary'`. Filters are preserved when navigating between pages.
```
Comment (ally-sassman) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx:157]: @jack-berg Would "**Sort by**" be better than "**Narrow data to..**"? When clicking around in a Demogorgon app, I don't see the "Narrow data to" option at the top of any APM UI pages.

Comment (jack-berg) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx:157]: I'm not familiar with the Demogorgon app, but the UI component I'm referencing does indeed use the language "Narrow data to". See screenshot:
Screenshot 2024-05-24 at 10 02 33 AM
Let's sync up to take a look at Demogorgon to make sure we're looking at the same type of app.

Comment (ally-sassman) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx:157]: Oops, I was not looking at an OTEL service. Thanks for the screenshot!

**Review by ghost — APPROVED** [review]

**Pull request merged**